vmdb/spec/controllers/catalog_controller_spec.rb

```diff
@@ -345,7 +345,8 @@
 
   context "#service_dialog_create_from_ot" do
     before(:each) do
-      @ot = FactoryGirl.create(:orchestration_template_with_content)
+      file = 'spec/fixtures/orchestration_templates/cfn_parameters.json'
+      @ot = FactoryGirl.create(:orchestration_template_cfn, :content => IO.read(file))
```

Comment:
Can it just be in the factory itself?  Also, personally I prefer File.read over IO.read.